File: SoloLearn/Code Coach/Snap, Crackle and Pop/SnapCracklePop.java
import java.util.Scanner;

public class SnapCracklePop {

    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int number;
        for (int i=0; i<6; i++) {
            number = input.nextInt();
            if (number % 3 == 0) 
                System.out.print("Pop ");
            else if (number % 2 == 0)
                System.out.print("Crackle ");
            else
                System.out.print("Snap ");
        }
        input.close();
    }
}

File: SoloLearn/Code Coach/Snap, Crackle and Pop/SnapCracklePop.cpp
#include <iostream>
using namespace std;

int main() {
   int number;
    for (int i=0; i<6; i++) {
        cin >> number;
        if (number % 3 == 0) 
            cout << "Pop ";
        else if (number % 2 == 0)
            cout << "Crackle ";
        else
            cout << "Snap ";
    }
    return 0;
} 

